corehq/apps/reports/static/reports/js/report_config_models.js

```diff
@@ -164,7 +164,7 @@ hqDefine("reports/js/report_config_models", [
         // edit the config currently being viewed
         self.setConfigBeingEdited = function (config) {
             var filters = {},
-                excludeFilters = ['startdate', 'enddate', 'format', 'date'];
+                excludeFilters = ['startdate', 'enddate', 'format'];
```

Comment:
Do we know why this was excluded originally?

Reply:
Unfortunately, I was not able to find out the reason. This was done years back. 

One thing I noted was  - `startdate` and `enddate` are excluded because you have an feature in UI itself while saving to select this. This also allows further edits to the saved report. The code has corresponding logic in place that deals with this.

No such feature or logic exists for `date` (single date filter) and so should not have been excluded originally.
This was probably not noticed earlier because the single date filter use is rare - only 1 report currently uses this.